src/Ztm.Data.Entity.Postgres/ByteaHandler.cs

```diff
@@ -1,66 +1,34 @@
 using System;
-using System.Text;
+using System.Threading.Tasks;
 using NBitcoin;
 using Npgsql;
 using Npgsql.BackendMessages;
 using Npgsql.TypeHandling;
 
 namespace Ztm.Data.Entity.Postgres
 {
-    class ByteaHandler :  NpgsqlSimpleTypeHandlerWithPsv<byte[], uint256>
+    class ByteaHandler : Npgsql.TypeHandlers.ByteaHandler, INpgsqlTypeHandler<uint256>
     {
-        internal readonly bool storeUInt256AsLE;
-
-        public ByteaHandler(bool lendian = false)
-        {
-            storeUInt256AsLE = lendian;
-        }
-
-        #region Read
-        public override byte[] Read(NpgsqlReadBuffer buf, int len, FieldDescription fieldDescription = null)
+        async ValueTask<uint256> INpgsqlTypeHandler<uint256>.Read(NpgsqlReadBuffer buf, int len, bool async, FieldDescription fieldDescription)
         {
             try
             {
                 var bytes = new byte[len];
-                buf.ReadBytes(bytes, 0, len);
-                return bytes;
+                await buf.ReadBytes(bytes, 0, len, async);
+                return new uint256(bytes, false);
```

Comment:
Currently we store `uint256` in the database as little endian, so this one is not correct that passing second parameter as false.